ui/tests/pages/components/json-editor.js

```diff
@@ -10,4 +10,8 @@ export default {
   hasToolbar: isPresent('[data-test-component=json-editor-toolbar]'),
   hasJSONEditor: isPresent('[data-test-component="code-mirror-modifier"]'),
   canEdit: notHasClass('readonly-codemirror'),
+  readOnlyTitle: text('.hds-code-block__title'),
+  readOnlyToolbar: isPresent('.hds-code-block__header'),
+  readOnlyCopyButton: isPresent('.hds-code-block__copy-button'),
+  readOnlyDisplay: isPresent('.hds-code-block__code'),
```

Comment:
I think that we're trying to move away from using page objects -- specifically instances of `ember-cli-page-object`; although I don't specifically remember why.. I don't think the library is deprecated, but @hellobontempo probably knows why.